index.html

```diff
@@ -7,195 +7,157 @@
+              <ul class="c-b-2015-ul">
+                <li><a href="#" class="canopy">CANOPY</a></li>
+                <li><a href="#"><img src="img/dot.svg" alt="dot"></a></li>
+                <li><a href="#" class="bed">Back End Dev</a></li>
+                <li><a href="#"><img src="img/dot.svg" alt="dot"></a></li>
+                <li><a href="#" class="year">2015</a></li>
+              </ul></div>
+              <div><p class="a-day">A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+              <ul class="prog-lang">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
                 </ul>
-                
-              </div>
-    <div><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
-              
+            <div><a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a></div></div>
+
+            <div class="items-3" id="work-1">
+              <div class="portofolio-img"><img class="portofolio-img" src="img/portofolio-img-3.svg" alt="my tonic project"></div>
+              <div class="item-2"><h5>Tonic</h5></div>
+              <div class="c-b-2015">
+              <ul class="c-b-2015-ul">
+                <li><a href="#" class="canopy">CANOPY</a></li>
+                <li><a href="#"><img src="img/dot.svg" alt="dot"></a></li>
+                <li><a href="#" class="bed">Back End Dev</a></li>
+                <li><a href="#"><img src="img/dot.svg" alt="dot"></a></li>
+                <li><a href="#" class="year">2015</a></li></ul></div>
+            <div><p class="a-day">A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
               <ul class="prog-lang">
                 <li>html</li>
                 <li>css</li>
                 <li>javaScript</li>
               </ul>
-              <a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a>
-            </div>
-              <div class="items">
-                <div class="portofolio-img-2"></div>
-                <h2 class="project-title">Multi-Post <br> Stories</h2>
-                  <div class="c-b-2015">
-                  <h6>CANOPY</h6>
-                  <ul class="c-b-2015-ul">
-                    <li>Back End Dev</li>
-                    <li>2015</li>
-                  </ul>
-                </div>
-      <div><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
-      
-                <ul class="prog-lang">
-                  <li>html</li>
-                  <li>css</li>
-                  <li>javaScript</li>
-                </ul>
-                <a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a>
-              </div>
+      <div><a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a></div></div>

```

Comment:
i suggest u to always use class on div because it is helpful to recognize in one glance